routes/api/items.js

```diff
@@ -0,0 +1,42 @@
+const express = require( 'express' )
+const router = express.Router()
+
+const { allItemsQuery, filteredItemsQuery, respondWithItemsApi } = require( '../items/item_response' )
+const { buildTree } = require( '../items/tree_creation' )
+
+router.get( '/list/:user_id/:email', ( request, response ) => {
```

Comment:
See comments above; I think we should consider different names for these that map to the entities in our domain